R/labels.r

```diff
@@ -51,10 +55,20 @@ update_labels <- function(p, labels) {
 #' # The plot tag appears at the top-left, and is typically used
 #' # for labelling a subplot with a letter.
 #' p + labs(title = "title", tag = "A")
-labs <- function(...) {
+#'
+#' # If you want to remove a label, set it to NULL.
+#' p + labs(title = "title") + labs(title = NULL)
+labs <- function(..., title, subtitle, caption, tag) {
   args <- list(...)
-  if (is.list(args[[1]])) args <- args[[1]]
+  if (length(args) > 0 && is.list(args[[1]])) args <- args[[1]]
   args <- rename_aes(args)
+
+  # Since NULL should be preserved, we should wrap args with list()
+  if (!missing(title)) args["title"] <- list(title)
```

Comment:
@lionel- can you think of a more elegant way to write this?

Reply:
Perhaps remove the named arguments and let users pass them through `...`?

Reply:
Hmm I can't think of anything better.

Reply:
Thanks, then I leave this as is :)